lightly/__init__.py

```diff
@@ -123,12 +123,11 @@ def _is_prefetch_generator_available():
     from lightly import transforms
     from lightly import utils
     
-    from lightly.api.version_checking import do_version_check
-
     if os.getenv('LIGHTLY_DID_VERSION_CHECK', 'False') == 'False':
         os.environ['LIGHTLY_DID_VERSION_CHECK'] = 'True'
 
         try:
-            do_version_check(current_version=__version__)
+          from lightly.api.version_checking import do_version_check
+          do_version_check(current_version=__version__)
         except Exception as e:
```

Comment:
Let's also fix the root of the problem and catch the specific exception we care about.

Reply:
Why did you move the import inside of the try-block?

Reply:
How can the import fail?

Reply:
I moved the import outside again 👍 my comment was about the type of exceptions we catch